CODEOWNERS

```diff
@@ -2,3 +2,4 @@
 
 # Fabric Maintainers
 *       @hyperledger/fabric-core-maintainers
+/docs/  @fabric-core-doc-maintainers
```

Comment:
this may need to be @hyperledger/fabric-core-doc-maintainers

Reply:
Updated.